framework/source/class/qx/test/Mixin.js

```diff
@@ -240,8 +240,8 @@ qx.Class.define("qx.test.Mixin",
           {
             var s = "";
             if (!dontRecurs) {
-              this.__b = new qx.Patch2();
-              s += "++" + this.__b.foo(true) + "__";
+              this['__b'] = new qx.Patch2();
```

Comment:
That's a problem. __b is a private variable. Private variables are not accessible even to mixins, IIRC. The __b in MPatch is different than the __b in Patch1.